File: files/readFile.java
package springIntro;

import java.io.*;

public class readFile {

	public static void main(String[] args) throws IOException{
		int i;
		File file = new File("C:\\Users\\Seyrek\\eclipse-workspace\\springIntro\\src\\springIntro\\text.txt");
		FileInputStream read = new FileInputStream(file);
		do {
			i = read.read();
			if(i != -1)
				System.out.print((char) i);
		}while(i != -1);
		read.close();
	}

}

File: files/readFile.cpp
#include<iostream>
#include<fstream>
using namespace std;
int main(){
	ifstream read("C:\\Users\\Seyrek\\Desktop\\C++\\files\\text.txt");
	string str;
	while(! read.eof()){
		str = read.get();
		if(read.eof())
			break;
		cout << str;
	}
	read.close();
	return 0;
}
